run_experiment.py

```diff
@@ -0,0 +1,163 @@
+# flake8: noqa
```

Comment:
Why not qa this script? (doens't have to, just curios)

Reply:
Also i think this would be better in a seperate folder e.g. `scripts` or `experiments` so its clear is seperate from the python package

Reply:
> Why not qa this script? (doens't have to, just curios)

I had problems with the warning filter in line 7, if I put it as flake8 requested, the warning wouldn't be filtered 😄 

> Also i think this would be better in a seperate folder e.g. scripts or experiments so its clear is seperate from the python package

Right, will put this in an experiment folder.